.gitattributes

```diff
@@ -0,0 +1,2 @@
+* text eol=crlf
```

Comment:
The second line specifies the local method for line endings as opposed to the method used in the repository. Feel free to remove or to change to auto or ignore the file fully. Just used it to get the other files in line with your preferred style.